node/Cargo.toml

```diff
@@ -3,67 +3,82 @@ authors = ["Eliott Teissonniere <git.eliott@teissonniere.org>"]
 build = "build.rs"
 edition = "2018"
 name = "nodle-chain"
-version = "2.0.13"
+version = "2.0.12"
+
+[package.metadata.wasm-pack.profile.release]
+# `wasm-opt` has some problems on linux, see
+# https://github.com/rustwasm/wasm-pack/issues/781 etc.
```

Comment:
I appreciate this note about the process and reasoning for setting this option. 💯